projects/fsastorefrontstyles/scss/base/building-blocks.scss

```diff
@@ -199,10 +199,20 @@
     text-align: center;
   }
 }
-.vertical-center-left {
```

Comment:
No need for `vertical-center-left` class anymore. I made `vertical-center` class global, with utile classes (top, bottom, right, left) we can easily change position, and also we have our global class `z-index-1`.